pages/_app.js

```diff
@@ -1,14 +1,36 @@
 import GlobalStyle from "@/styles";
-import Head from "next/head";
+
+import styled from "styled-components";
+import Donations from "../db.json";
```

Comment:
Hey Mina, I like our code so far but please use a lowercase name, Capital names are reserved for react components and classes.